lib/sync-settings.coffee

```diff
@@ -1,9 +1,8 @@
 # imports
 {BufferedProcess} = require 'atom'
-GitHubApi = require 'github'
-_ = require 'underscore-plus'
-PackageManager = require './package-manager'
 fs = require 'fs'
+_ = require 'underscore-plus'
+[GitHubApi, PackageManager] = []
```

Comment:
It does that so Atom doesn't have to read the module files when it starts up. It reduced package load time by 10-20ms on my machine (according to timecop).